README.rst

```diff
@@ -106,6 +106,19 @@ For developers that want to ease the translation process, please read `this docu
 Building
 --------
 
+Nightly Automated Build Steps
+=============================
+
+1. **Fetch sources**
+2. ``git clone https://github.com/nextcloud/documentation.git``
+3. ``cd documentation``
+4. ``git checkout $1``
```

Comment:
is `$1` an argument of a script?

Reply:
this is going to be the name of the target branch
I will modify it to be easier to read